web/templates/web/_navigation.html

```diff
@@ -1,23 +1,70 @@
 {% load i18n %}
 {% load web_extras %}
 {% trans "Logout" as logout %}
-<header class="row">
-    <nav class="nav">
-        {% nav_link request 'web:home' 'Home' %}
-        {% if user.is_researcher %}
-            <a class="nav-link" href="{% url 'exp:study-list' %}">{% trans "Experimenter" %}</a>
-        {% endif %}
-        {% nav_link request 'web:studies-list' 'Studies' %}
-        {% nav_link request 'web:faq' 'FAQ' %}
-        {% nav_link request 'web:scientists' 'The Scientists' %}
-        {% nav_link request 'web:resources' 'Resources' %}
-        {% if user.is_authenticated %}
-            {% nav_link request 'accounts:manage-account' 'My Account' %}
-            {% nav_link request 'web:studies-history' 'My Past Studies' %}
-            {% nav_link request 'logout' logout %}
-        {% else %}
-            {% nav_signup request %}
-            {% nav_login request %}
-        {% endif %}
+<style>
+
+    ul.navbar-nav li.nav-item button.active,
+    ul.navbar-nav li.nav-item a.active {
+        --bs-nav-pills-link-active-bg: #e7e7e7;
+    }
+
+</style>
```

Comment:
Let's move this into a css file please.

Reply:
@okaycj sure! Does that mean I should add the `base.css` file back in? Or are we doing separate css files that correspond to html files?

Reply:
FYI I put this in a new CSS file: `web/static/css/navigation.css` and loaded the file in `base.html`.